third_party/move/move-binary-format/src/file_format_common.rs

```diff
@@ -59,6 +59,7 @@ pub const BYTECODE_INDEX_MAX: u64 = 65535;
 pub const LOCAL_INDEX_MAX: u64 = 255;
 
 pub const IDENTIFIER_SIZE_MAX: u64 = 65535;
```

Comment:
Can you rename this to something like "LEGACY_IDENTIFIER_SIZE_MAX"? 

Would be even better if we could mark it as deprecated. Not sure if this will work though.

Reply:
Done renaming!

I think we can deprecate consts. Perhaps, we should do it after the feature is enabled in every network?